src/firefly/js/templates/lightcurve/wise/WiseMissionOptions.js

```diff
@@ -29,46 +32,85 @@ export class WiseSettingBox extends Component {
 
         const wrapperStyle = {margin: '3px 0'};
 
+        const tblModel = getTblById(LC.RAW_TABLE);
+        //const validTimes = get(missionEntries, LC.META_FLUX_NAMES, []);
+        //const validValues = get(missionEntries, LC.META_FLUX_NAMES, []);
+
+        var getList = (val, type) => {
+            var colType = (!type || type === 'numeric') ?
+                ['double', 'd', 'long', 'l', 'int', 'i', 'float', 'f'] : ['char', 'c', 's', 'str', 'double', 'd', 'long', 'l', 'int', 'i', 'float', 'f'];
+
+            return get(tblModel, ['tableData', 'columns']).reduce((prev, col) => {
+                if ((colType.includes(col.type)) &&
+                    (!has(col, 'visibility') || get(col, 'visibility') !== 'hidden') &&
+                    (col.name.startsWith(val))) {
+                    prev.push(col.name);
+                }
+                return prev;
+            }, []);
+        };
         var allCommonEntries = Object.keys(generalEntries).map((key) =>
             <ValidationField key={key} fieldKey={key} wrapperStyle={wrapperStyle}
                              style={{width: 80}}/>
         );
 
         const missionKeys = [LC.META_TIME_CNAME, LC.META_FLUX_CNAME];
         const missionOtherKeys = [LC.META_ERR_CNAME];
+        const imageDataSource = [LC.META_URL_CNAME]; // use meta_url from generic case, see LcManager.js:320 for why
         const missionListKeys = [LC.META_TIME_NAMES, LC.META_FLUX_NAMES];
-
+        const topZ = 9999;
```

Comment:
This is problematic.  9999 is too high.  The validation error message for the same field is showing underneath the suggestion box.